src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,25 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import cn from 'classnames';
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article className={cn(
+      'TodoInfo', {
+        'TodoInfo--completed': completed,
+      },
+    )}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+      {user
+        ? <UserInfo user={user} />
+        : 'User not found'}
```

Comment:
```suggestion
      <h2 className="TodoInfo__title">
        {title}
      </h2>
      
      {user
        ? <UserInfo user={user} />
        : 'User not found'
      }
```

Reply:
Last changes added in [add task solution] commit